source/fuzz/transformation_replace_id_with_synonym.cpp

```diff
@@ -140,6 +140,51 @@ bool TransformationReplaceIdWithSynonym::IsAgnosticToSignednessOfOperand(
     case SpvOpUGreaterThanEqual:
     case SpvOpSGreaterThanEqual:
       return true;
+
+    case SpvOpAtomicStore:
+    case SpvOpAtomicExchange:
+    case SpvOpAtomicIAdd:
+    case SpvOpAtomicISub:
+    case SpvOpAtomicSMin:
+    case SpvOpAtomicUMin:
+    case SpvOpAtomicSMax:
+    case SpvOpAtomicUMax:
+    case SpvOpAtomicAnd:
+    case SpvOpAtomicOr:
+    case SpvOpAtomicXor:
+    // TBD(To Be Discussed),
+    // https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/EXT/SPV_EXT_shader_atomic_float_add.asciidoc#modifications-to-the-spir-v-specification-version-15
+    case SpvOpAtomicFAddEXT:  // Capability (AtomicFloat32AddEXT,
+                              // AtomicFloat64AddEXT)
+      assert(use_in_operand_index != 0 && "Forbidden.");
```

Comment:
Change "Forbidden" to something like: "Signedness check should not occur on a pointer operand."

Reply:
Done.